run_dir/static/js/running_notes.js

```diff
@@ -20,7 +20,7 @@ function generate_category_label(category){
          category='<span class="label label-warning">'+ category +"</span>";
      }else if (category == 'Important'){
          category='<span class="label label-imp">'+ category +"</span>";
-     }else if (category == 'Deviation'){	
+     }else if (category == 'Deviation'){
```

Comment:
I choose not to see that you've added this file change

Reply:
Atom added it to the file and I added it to the PR by mistake 😅